src/ApiCalls.js

```diff
@@ -0,0 +1,50 @@
+import axios from "axios"
+
+const movie = "https://api.themoviedb.org/3/";
```

Comment:
Это явные константы, и во всех языках программирования принято называть их в верхнем регистре
```suggestion
const MOVIE = "https://api.themoviedb.org/3/";
```